app/assets/stylesheets/dashboard.scss

```diff
@@ -231,7 +231,7 @@
     form {
       padding: 10px 0px;
       font-size: 15px;
-      font-family: $helvetica-condensed;
+      font-family: inherit;
```

Comment:
I think instead of changing this to `inherit`, we could actually just delete the line completely 🤔

Reply:
Yup makes sense @aitchiss , removing it altogether yields same result. Updated new changes 👍